doc/additional-configurations.md

```diff
@@ -127,7 +136,7 @@ Before you get to the task of creating a new database for SonarQube, you need to
 		![](_img/New-Database-User-Mapping.png)
 	- Click **OK** to complete the new user setup.
 
-3. **Test connection**
+4. **Test connection**

```

Comment:
Updated. I have chosen to keep SQL User creation as Step 3 and marked it as optional.